src/browser/menu.js

```diff
@@ -179,6 +180,10 @@ class Menu {
           }
           break
 
+        case 'export':
+          item = buildExportMenu(item, params, this.responder.bind(this))
```

Comment:
Let's move the menu building logic here; the plugin system does not need to know about menu building internals.